demo/common/overview.c

```diff
@@ -584,6 +584,36 @@ overview(struct nk_context *ctx)
                         nk_combo_end(ctx);
                     }
                 }
+                {
+                    /* Combobox from array of struct */
+                    static int sel=0;
+                    struct student {
+                        int id;
+                        char* name;
+                        int age;
+                        char* major;
+                    };
```

Comment:
May want to declare the struct outside of the code block.

Reply:
Declare or define? What would be the reasoning g for this? I have definitely seen structure definitions with local scope. 

But then again. I don't think that is a c89 capability.